### PR TITLE
sql: add user statements to database test

### PR DESCRIPTION
**Pull request by madelynnblue**
This is currently completely wedging a server in just a few seconds.

Add some RSG plumbing to make "is the server wedged or is this just a
slow statement" detection much easier.

Release note: None

### CODE REVIEW

**Comment by mjibson** [comment]
make test 'PKG=./pkg/sql/tests' 'TESTS=TestRandomSyntaxSchemaChangeDatabase' 'TESTFLAGS=-v -rsg 1m -rsg-routines 8 -rsg-exec-timeout 5s'

**Comment by mjibson** [comment]
[goroutine.txt](https://github.com/cockroachdb/cockroach/files/2329776/goroutine.txt)

File: pkg/sql/tests/rsg_test.go [hunk 1]
```diff
@@ -124,7 +124,30 @@ func (db *verifyFormatDB) exec(ctx context.Context, sql string) error {
...
 		fmt.Printf("%s\n", b[:n])
-		panic(errors.Errorf("timeout: %q. currently executing: %v", sql, db.mu.active))
+		// Now see if we can execute a SELECT 1. This is useful because sometimes an
+		// exec timeout is because of a slow-executing statement, and other times
+		// it's because the server is completely wedged. This is an automated way
+		// to find out.
+		errch := make(chan error, 1)
+		go func() {
+			rows, err := db.db.Query(`SELECT 1`)
+			if rows != nil {
```
Comment (vivekmenezes) [inline pkg/sql/tests/rsg_test.go]: if err == nil {
rows.Close()
}

**Review by vivekmenezes — APPROVED** [review]

**Comment by knz** [comment]
this will want a backport I think

**Comment by mjibson** [comment]
done

**Pull request merged**